packages/blade/src/components/Tooltip/_decisions/decisions.md

```diff
@@ -0,0 +1,107 @@
+# Tooltip API Decisions <!-- omit in toc -->
+
+The tooltip typically provides additional context about the element or its function. A tooltip is always triggered by a `mouse hover` on desktop and `on tap` on mobile.
+
+<img src="./tooltip-thumbnail.png" width="380" alt="" />
+
+- [Design](#design)
+- [`Tooltip` API](#tooltip-api)
+  - [Examples:](#examples)
+    - [Basic](#basic)
+- [Library](#library)
+- [Motion](#motion)
+- [Accessibility](#accessibility)
+- [Open Questions](#open-questions)
+
+## Design
+
+[Figma Link](https://www.figma.com/file/jubmQL9Z8V7881ayUD95ps/Blade---Payment-Light?type=design&node-id=40540-559304&t=tmTrf3xJU6oj59fM-0) to all variants of the Tooltip component
+
+## `Tooltip` API
+
+| Prop          | Type                                                                     | Default     | Description                                                               | Required |
+| ------------- | ------------------------------------------------------------------------ | ----------- | ------------------------------------------------------------------------- | -------- |
+| content       | `string`                                                                 | `undefined` | Content of the tooltip                                                    | ✅       |
+| placement     | `top, top-start, top-end, left, right, bottom, bottom-start, bottom-end` | `top`       | Placement of tooltip                                                      | ✅       |
```

Comment:
Do we need this if we expose styled props on Tooltip?

We can have a default position and consumers can position the tooltip the way they like with

```jsx
<Tooltip position="absolute" left="" right="" top="" bottom="" />
```

understandable that its a little bit more work but will be consistent and might be intuitive since this what they do for other components as well

Reply:
I don't think this position would work, `position` !== `placement`. 
Placement is automatically computed. while position is something the user will have to maintain. 

The placement is a suggestion, if user says `placement="top"` it might not always be on top if the space isn't there for it to be. it can automatically move to "bottom". 

Plus if we let consumers position the tooltip they will inevitability mess up the spacing between the the trigger and the tooltip content resulting in inconsistencies.